src/Microsoft.Azure.WebJobs.Logging.ApplicationInsights/DefaultTelemetryClientFactory.cs

```diff
@@ -128,23 +144,19 @@ internal TelemetryConfiguration InitializeConfiguration()
         /// <returns>The <see cref="ITelemetryChannel"/></returns>
         protected virtual ITelemetryChannel CreateTelemetryChannel()
         {
+#if !NETSTANDARD2_0
             return new ServerTelemetryChannel();
-        }
-
-        /// <summary>
-        /// Creates the <see cref="QuickPulseTelemetryModule"/> to be used by the <see cref="TelemetryClient"/>.
-        /// </summary>
-        /// <returns>The <see cref="QuickPulseTelemetryModule"/>.</returns>
-        protected virtual QuickPulseTelemetryModule CreateQuickPulseTelemetryModule()
-        {
-            return new QuickPulseTelemetryModule();
+#else
+            throw new NotSupportedException();
```

Comment:
Good to know. The AI stuff is the key area I was hoping you'd get your eyes on. I'll update this.